.github/workflows/check-links.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: '0.96.0'
+          hugo-version: '0.111.0'
```

Comment:
Upgrading Hugo to use its `Ancestors` feature, which makes breadcrumbing _way_ easier.